src/wagtail_factories/factories.py

```diff
@@ -4,19 +4,17 @@
 from django.utils.text import slugify
 from factory import errors, utils
 from factory.declarations import ParameteredAttribute
-from wagtail import VERSION as WAGTAIL_VERSION
-from wagtail.core.models import Collection, Page, Site
 from wagtail.images import get_image_model
 
 try:
-    from factory.django import DjangoModelFactory
+    from wagtail.models import Collection, Page, Site
 except ImportError:
-    from factory import DjangoModelFactory
+    # Wagtail<3.0
+    from wagtail.core.models import Collection, Page, Site
 
-if WAGTAIL_VERSION >= (2, 8):
```

Comment:
Drop support where WAGTAIL_VERSION<2.15

Reply:
**Question**: I guess the conditional is no longer needed here, just checking 😄

Reply:
Yes, that's why I removed it entirely.
I like to leave some notes for people to see what I am doing just in case.